doc/classes/InputEventJoypadButton.xml

```diff
@@ -16,8 +16,9 @@
 		<member name="pressed" type="bool" setter="set_pressed" getter="is_pressed" default="false">
 			If [code]true[/code], the button's state is pressed. If [code]false[/code], the button's state is released.
 		</member>
-		<member name="pressure" type="float" setter="set_pressure" getter="get_pressure" default="0.0">
+		<member name="pressure" type="float" setter="set_pressure" getter="get_pressure" default="0.0" is_deprecated="true">
 			Represents the pressure the user puts on the button with their finger, if the controller supports it. Ranges from [code]0[/code] to [code]1[/code].
```

Comment:
Why keep the description if it's actually untrue?

Reply:
It still represents it. If somehow an user implemented it, that's what "pressure" is for. But it could be cut down significantly, I suppose. The implementation details do not matter if there's no implementation to speak of 🙃 .